.github/workflows/esm-lint.yml

```diff
@@ -13,7 +13,7 @@ on:
       - '*'
   push:
     branches:
-      - master
+      - main
```

Comment:
is this duplicate now?

Reply:
Good catch, I initially manually removed it but then just redone everything with find and replace.

Reply:
https://github.com/fregante/github-url-detection/blob/ca4edcfdc87e26241c10fb27985d73af8d024630/.github/workflows/esm-lint.yml#L5